.ci/travis/travis.sh

```diff
@@ -115,6 +131,16 @@ jdk12-verify-limited)
   mvn -e verify -Dpmd.skip=true -Dspotbugs.skip=true
   ;;
 
+jdk13-assembly-site)
+  mvn -e package -Passembly
+  mvn -e site -Pno-validations
```

Comment:
it is the same as jdk12 ... do we need to remove jdk12 ? .... as we can not afford to expand CI a lot, it takes time.
I think we need only latest released, and we already discussed it.

Reply:
Sounds reasonable to me. We don't have Java-12 specific tests, so it can be safely dropped.